pkg/netpol/eval/internal/k8s/adminnetpol.go

```diff
@@ -234,6 +236,124 @@ func subjectSelectsPeer(anpSubject apisv1a.AdminNetworkPolicySubject, p Peer) (b
 	return doesPodsFieldMatchPeer(anpSubject.Pods, p)
 }
 
+// anpPortContains returns if the given AdminNetworkPolicyPort selects the input connection
+//
+//gocyclo:ignore
+func anpPortContains(rulePorts *[]apisv1a.AdminNetworkPolicyPort, protocol, port string, dst Peer) (bool, error) {
```

Comment:
does this function have some code used also in `func (np *NetworkPolicy) ruleConnsContain` that can be captured in helper functions instead?

Reply:
yes, done